.github/workflows/release.yml

```diff
@@ -0,0 +1,98 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      qemu_repo:
+        description: 'QEMU repo (e.g. https://github.com/qemu/qemu)'
+        required: false
+      qemu_version:
+        description: 'QEMU version (e.g. v5.2.0)'
+        required: false
+      target:
+        description: 'Bake target'
+        required: true
+        default: 'mainline'
+      latest:
+        description: 'Create latest tag'
+        required: true
+        default: 'false'
+      dry-run:
+        description: 'Dry run'
+        required: false
+        default: 'true'
+
+env:
+  REPO_SLUG: tonistiigi/binfmt
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Prepare
+        run: |
+          QEMU_REPO=$(docker buildx bake --print ${{ github.event.inputs.target }} | jq --raw-output '.. | .args?.QEMU_REPO | select(.)')
+          if [ "${{ github.event.inputs.qemu_repo }}" != "" ]; then
+            QEMU_REPO=${{ github.event.inputs.qemu_repo }}
+          fi
+          QEMU_VERSION=$(docker buildx bake --print ${{ github.event.inputs.target }} | jq --raw-output '.. | .args?.QEMU_VERSION | select(.)')
+          if [ "${{ github.event.inputs.qemu_version }}" != "" ]; then
+            QEMU_VERSION=${{ github.event.inputs.qemu_version }}
+          fi
+          QEMU_SHA=$(git ls-remote --sort='version:refname' ${QEMU_REPO} | grep ${QEMU_VERSION} | cut -f 1 | head -1)
```

Comment:
why is this needed?

Reply:
Yes it's not needed anymore since I've removed the `qemu_ref` input.